code/modules/admin/view_variables/view_variables_global.dm

```diff
@@ -29,14 +29,6 @@
 				"forumsqllogin",
 				"forumsqlpass",
 				"forumsqlport",
-				"sqladdress",
```

Comment:
why are these all deleted? we plan to use an sql database in future

Reply:
> why are these all deleted? we plan to use an sql database in future

Because Sqladress now in Config. No shitty changes